akka-actor-typed/src/main/scala/akka/actor/typed/ActorRef.scala

```diff
@@ -16,12 +16,10 @@ import scala.util.Success
  * Actor instance. Sending a message to an Actor that has terminated before
  * receiving the message will lead to that message being discarded; such
  * messages are delivered to the [[DeadLetter]] channel of the
- * [[EventStream]] on a best effort basis
+ * [[akka.typed.EventStream]] on a best effort basis
```

Comment:
that wrong package?

Reply:
Well i didn't move it it to akka.actor as the untyped EventStream isn't in there, it could be akka.event.typed.EventStream, WDYT?

Reply:
sounds good given the other package names

Reply:
Renamed